docs/OV_Runtime_UG/ShapeInference.md

```diff
@@ -39,6 +39,8 @@ There are other approaches to change model input shapes during the stage of <a h
 ### Dynamic Shape Notice
 
 Shape-changing functionality could be used to turn dynamic model input into a static one and vice versa.
+It is recommended to always set static shapes in case if the shape of data is not going to change from one inference to another.
```

Comment:
There is the same text on the 180-ish lines

Reply:
Why do we have the same notice in two places in the same document?

Reply:
Because of the Python/C++ switch